api/DOMParser.json

```
@@ -131,6 +131,48 @@
             "deprecated": false
           }
         },
+        "accepts_TrustedHTML": {
+          "__compat": {
+            "description": "Can be set with a `TrustedHTML` instance",
```

@caugner There was a post by @lukewarlow in https://github.com/mdn/content/issues/37518#issuecomment-2991844979 which we are best of to discuss here.

In summary, "accepts" is not quite right because actually you can pass anything that resolves to a string to any method that takes a string. This is why we can use the [tinyfill described here to fake TTs on systems that don't support them](https://developer.mozilla.org/en-US/docs/Web/API/Trusted_Types_API#trusted_types_tinyfill).
The thing that makes these methods special is that they _enforce_ TTs being passed when enforcement is enabled. 

While I don't think that most users will appreciate the difference, we would not be wrong to make it. This would would require changes to the other cases too.

Something like::

```suggestion
        "enforces_TrustedHTML": {
          "__compat": {
            "description": "Requires `TrustedHTML` instance when trusted types are enforced",
```
----

From Luke on other post


> I'm not sure where best to raise this as I can't file a raw issue on BCD I'll mention here.
> 
> I think the "accepts_TrustedHTML" style of subfeature in BCD may be the wrong way to frame what this data is for.
> 
> I think "enforces_TrustedTypes" might be a better key with something like "Must be set with a `TrustedHTML` instance when trusted types are enforced." (or something more succint)
> 
> Any API that accepts a string can be set to an TT object because they'll just get stringified by IDL type conversions.
> 
> That way stuff like Chromium's bug in `new Function()` can be reflected by a partial implemention and a note that TT is enforced for the sink but passing a TrustedScript doesn't actually work (it gets treated as an unsafe string and put through the default policy)

---

Yes, the new key approaches do reflect the intent. I agree with @lukewarlow suggestion https://github.com/mdn/browser-compat-data/pull/27114#issuecomment-3000135491 that it makes sense to use a common key everywhere and in the description specify the actual type. So updated suggestion reflecting this particular instance would be

```suggestion
        "enforces_trusted_types": {
          "__compat": {
            "description": "Requires `TrustedHTML` instance when trusted types are enforced",
```

Thanks for offering to do the PR @lukewarlow - I assume since we seem to be agreement with each other and @caugner you might reasonably start that.